crates/pgt_env/src/lib.rs

```diff
@@ -27,44 +28,67 @@ pub const VERSION: &str = match option_env!("PGT_VERSION") {
     },
 };
 
-pub static PGT_WEBSITE: &str = "https://pgtools.dev";
+pub static PGLS_WEBSITE: &str = "https://pgtools.dev";
 
-pub struct PgTEnv {
-    pub pgt_log_path: PgTEnvVariable,
-    pub pgt_log_prefix: PgTEnvVariable,
-    pub pgt_config_path: PgTEnvVariable,
+pub struct PgLSEnv {
+    pub pgls_log_path: PgLSEnvVariable,
+    pub pgls_log_level: PgLSEnvVariable,
+    pub pgls_log_prefix: PgLSEnvVariable,
+    pub pgls_config_path: PgLSEnvVariable,
+
+    // DEPRECATED
+    pub pgt_log_path: PgLSEnvVariable,
+    pub pgt_log_prefix: PgLSEnvVariable,
+    pub pgt_config_path: PgLSEnvVariable,
 }
 
-pub static PGT_ENV: OnceLock<PgTEnv> = OnceLock::new();
+pub static PGT_ENV: OnceLock<PgLSEnv> = OnceLock::new();
 
-impl PgTEnv {
+impl PgLSEnv {
     fn new() -> Self {
         Self {
-            pgt_log_path: PgTEnvVariable::new(
-                "PGT_LOG_PATH",
+            pgls_log_path: PgLSEnvVariable::new(
+                "PGLS_LOG_PATH",
                 "The directory where the Daemon logs will be saved.",
             ),
-            pgt_log_prefix: PgTEnvVariable::new(
-                "PGT_LOG_PREFIX_NAME",
+            pgls_log_level: PgLSEnvVariable::new(
```

Comment:
I think the `pgt` log level is missing, was that intended? not sure if anybody except for me uses it 😆